lib/embeds.js

```diff
@@ -106,6 +106,14 @@ function renderCustom (props) {
   return '';
 }
 
+function renderTumblr (props) {
+  const { url, text } = props;
+  const href = text[0] && text[0].href ? text[0].href : url;
+
+  // There is no other way to show tumblr posts currently
+  return <a target='_blank' class='tumblr-post' href={href}>Load Tumblr post in new window</a>;
```

Comment:
Thanks @iefserge, I wish there was but there is no post title in the embed code they supply only the url of the post so the only other option would be to make it the url. Below is a sample embed code:

```
<div class="tumblr-post" data-href="https://embed.tumblr.com/embed/post/xu6zPm7j0A8sEf03xcnR9w/101895859192" data-did="1332a9c3948cd691a2db1da3e68a58fbbf20e709" data-language="en_US"><a href="http://unwrapping.tumblr.com/post/101895859192/tumblr-sweatshirt-merch">http://unwrapping.tumblr.com/post/101895859192/tumblr-sweatshirt-merch</a></div>  <script async src="https://secure.assets.tumblr.com/post.js"></script>
```